### ISSUE TITLE
Sort album's songs by track number

### ISSUE THREAD

**Issue by deshpandetanmay03** [sosauce/CuteMusic#59]
Album songs are sorted by alphabetical order. this is not usable

sort them by track number metadata

**Comment by nubesurrealista** [comment]
First of all, **watch your tone.** This is a free and open source project, the dev **is not** an employee. Words like "**Please**" and "**Thank you**" can make a big difference.

Second, it's a known issue. The dev does what he can. 

Peace ✌️

**Comment by deshpandetanmay03** [comment]
pretty please

also if dev says this will not be implemented I'm fine with it. I'm not familiar with the code and I'm paying them. this is more of a suggestion if they are looking for one

**Comment by sosauce** [comment]
Yes, a please and thank you would've been appreciated, but it's okay for this time. Do you want to sort by track number on the main screen or album screen ? It's really not possible to sort by track number on the main screen as it contains all songs from any albums, thus could have the same track number !

**Comment by deshpandetanmay03** [comment]
album screen please

**Comment by sosauce** [comment]
Oh okay ! Will make it for next update ^^
